packages/next/src/build/webpack/plugins/terser-webpack-plugin/src/index.ts

```diff
@@ -147,6 +147,9 @@ export class TerserPlugin {
                 compress: true,
                 mangle: true,
                 module: 'unknown',
+                output: {
+                  comments: false,
+                },
```

Comment:
Eventually there will be no option to not strip the comments:

1. Since `swcMinify` will be true by default
2. User cannot turn it off
3. So terser `output.comment` is always false.

Did I understand it right?

Reply:
This isn't related to removing `swcMinify` config just making sure we're matching behavior which is to always strip all comments. It's a bug this behaves differently with swc currently.

Reply:
Thank you for explaining!